suitcase/fields.py

```diff
@@ -1057,6 +1057,15 @@ class UBInt64(BaseStructField):
     PACK_FORMAT = UNPACK_FORMAT = b">Q"
 
 
+class UBFloat32(BaseStructField):
```

Comment:
Sure thing. Is it alright if I added `Bool` (and possibly `String`) from this [table](https://docs.python.org/2/library/struct.html#format-characters) for completion sake?